<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchCalls</class>
 <widget class="QDialog" name="SearchCalls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Calls</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="pushButtonK">
     <property name="text">
      <string notr="true">K</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QPushButton" name="pushButtonE">
     <property name="text">
      <string notr="true">E</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QPushButton" name="pushButtonP">
     <property name="text">
      <string notr="true">P</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLineEdit" name="lineEditCalls"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="labelPossibleResults">
     <property name="text">
      <string>Possible Results:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="radioButtonElm">
     <property name="text">
      <string>Elm</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="radioButtonIrwin">
     <property name="text">
      <string>Irwin</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QLabel" name="labelKResponse">
     <property name="text">
      <string>K - I'm so glad you called! I was just about to call you, too! I guess we must be a good match!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QLabel" name="labelEResponse">
     <property name="text">
      <string>E - Hearing about your escapades rocks my soul! It sure does!</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="6">
    <widget class="QLabel" name="labelPResponse">
     <property name="text">
      <string>P - How are you? What are you doing? Where are you? How many Badges do you have now? How much money have you saved? How's your mom? Have you got lots of Pok√©mon? Is it going to be sunny tomorrow? Arrgh, there's so much I want to chat about! This is going nowhere!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="pushButtonOkay">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
